src/cargo_shim/mod.rs

```diff
@@ -525,7 +525,24 @@ impl BuildTarget {
 #[derive(Copy, Clone, PartialEq, Debug)]
 pub enum MessageFormat {
     Human,
-    Json
+    Json,
+    #[doc(hidden)]
+    Other,
```

Comment:
Rename `Other` to `__Nonexhaustive` as it's usually done (so that there will be no doubt that this is just a dummy variant).